mithril-aggregator/src/certifier_service.rs

```diff
@@ -58,6 +58,20 @@ pub enum CertifierServiceError {
     /// Codec error.
     #[error("codec error: '{0}'")]
     Codec(String),
+
+    /// No certificate for this epoch
+    #[error("There is an epoch gap between the last certificate epoch ({certificate_epoch:?}) and current epoch ({current_epoch:?})")]
+    CertificateEpochGap {
+        /// Epoch of the first certificate found
```

Comment:
Maybe more clear to avoid confusion between first /last certificates
```suggestion
        /// Epoch of the last certificate
```